client/wfm/src/process-instance/ProcessInstanceRepository.ts

```diff
@@ -0,0 +1,8 @@
+import {ProcessInstance} from './ProcessInstance';
+export interface ProcessInstanceRepository {
+  getAll(): Promise<ProcessInstance[]>;
```

Comment:
I think we should include the pagination stuff in the JIRAs for the store. +1 on adding the TODO comment though.